### DISCUSSION TITLE
Adding custom plugin in desktop jbrowse2

### DISCUSSION THREAD

**Discussion by nponts** [GMOD/jbrowse-components#3300] [Q&A]
Hi,
I would like to add one of those so many plugins I found on github in my jbrowse2 desktop. How do I do that?
Cheers,

NP

**Comment by garrettjstevens** [comment]
Which plugin do you want to install? Plugins that have been verified can be installed through the [plugin store](https://jbrowse.org/jb2/docs/user_guides/plugin_store/).
